checks/__init__.py

```diff
@@ -392,17 +393,26 @@ def _get_check_manifest(self, manifest_path=None):
             CheckClass.__init__() -> AgentCheck.__init__() -> AgentCheck._get_check_manifest()
         """
         from checks.network_checks import NetworkCheck
+        try:
+            from checks.wmi_check import WinWMICheck
```

Comment:
Yeah, since we have to set WInWMICheck to `None` otherwise this seemed to be a pattern that worked well.